### PR TITLE
#48in24: anagram

### PR DESCRIPTION
**Pull request by fejan-malek**
https://github.com/exercism/php/issues/734

-  Change: Added all the missing `testcases`
- Also removed unused `testcases`.

### CODE REVIEW

**Comment by mk-mxp** [comment]
@fejan-malek You were too early with this one. 2024-07-23 minus 2 weeks = 2024-07-09. Please do not haste forward, it causes extra work.

I'll leave this opened. On 2024-07-09 (or some days after) please re-run the `configlet` sync command and rebase this branch. Thanks for your patience!

**Review by mk-mxp — CHANGES_REQUESTED** [review]
Thanks for the nice work! You are helping very much.

File: exercises/practice/anagram/AnagramTest.php [hunk 1]
```diff
@@ -31,108 +9,162 @@ public static function setUpBeforeClass(): void
         require_once 'Anagram.php';
     }
 
+    /**
+     * @testdox no matches
+     * uuid dd40c4d2-3c8b-44e5-992a-f42b393ec373
```
[the remark this answers is not in the document]
Comment (fejan-malek) [reply exercises/practice/anagram/AnagramTest.php:14]: I will keep it in mind. But past exercises are synced already in the same order of `@testdox` and `uuid`.

If we need to update that also then I will create a separate pull request.

[the remark this answers is not in the document]
Comment (mk-mxp) [reply exercises/practice/anagram/AnagramTest.php:14]: The past ecxercises were done in #773, thank you.

**Review by mk-mxp — APPROVED** [review]
Thanks a lot for helping with this one!

**Pull request merged**